core/src/test/java/org/apache/calcite/test/RexProgramTest.java

```diff
@@ -1581,6 +1588,13 @@ private void checkExponentialCnf(int n) {
     final RexLiteral literal3 = rexBuilder.makeExactLiteral(BigDecimal.valueOf(3));
     final RexLiteral literal4 = rexBuilder.makeExactLiteral(BigDecimal.valueOf(4));
 
+    // "a <> 1 or a <> 2" ==> "true"
+    checkSimplifyFilter(
```

Comment:
I know we have a lot of these "big" multi-testcase tests in rexprogramtest; but if possible - could you please organize the new testcases to be individually executable - it's so much easier to debug a single testcase in case something breaks...

Reply:
Ok, I'll re-organize the test